### PR TITLE
docs: Update the component usage guide to include another callback method

### PR DESCRIPTION
**Pull request by Kigstn**
## What type of pull request is this?

- [ ] Non-breaking code change
- [ ] Breaking code change
- [x] Documentation change/addition

## Description

Update the component usage guide to include another callback method

## Changes

- add guide for `@component_callback(custom_id)`

## Checklist

- [ ] I've formatted my code with [Black](https://black.readthedocs.io/en/stable/)
- [ ] I've ensured my code works on `Python 3.10.x`
- [ ] I've tested my code

### CODE REVIEW

File: docs/src/Guides/5 Components.md [hunk 1]
```diff
@@ -201,10 +201,37 @@ When responding to a component you need to satisfy discord either by responding
...
+                    style=ButtonStyles.GREEN,
+                    label="Click Me",
+                    disabled=False,
+                )
+            )
+        ]
+
+        await channel.send("Look a Button!", components=components)
+
+    @component_callback("my_button")
```
Comment (silasary) [inline docs/src/Guides/5 Components.md]: I'd put comment here mentioning that the parameter is the parameter is the custom ID

Comment (benwoo1110) [reply docs/src/Guides/5 Components.md]: maybe to be clearer, also change `"my_button"` to `"my_button_id"`

**Review by benwoo1110 — APPROVED** [review]
looks good

**Comment by artem30801** [comment]
Looks good, but for simplicity, I'd advise pointing out that you can actually pass just one component or actionrow to `components` kwarg. Would also make code a LOT less scary in all examples

**Comment by Kigstn** [comment]
> Looks good, but for simplicity, I'd advise pointing out that you can actually pass just one component or actionrow to `components` kwarg. Would also make code a LOT less scary in all examples

Fair, I'll update that

**Comment by artem30801** [comment]
Nice, thank you
One more question - why do all the examples use `disabled=False` in Button creation? I don't think it's necessary in most cases

**Comment by Kigstn** [comment]
> Nice, thank you One more question - why do all the examples use `disabled=False` in Button creation? I don't think it's necessary in most cases

Good point!

**Review by artem30801 — APPROVED** [review]
LGTM

**Review by silasary — APPROVED** [review]
Yeah, I'm happy with this.

But if I were to nitpick, I'd point out that Persistent Callback Option 3 should use an Action Row, because the complexity of that option is only valuable when you have multiple buttons

**Comment by Kigstn** [comment]
> But if I were to nitpick, I'd point out that Persistent Callback Option 3 should use an Action Row, because the complexity of that option is only valuable when you have multiple buttons

While I agree, I would keep it this way for consistency sake

**Pull request merged**